app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,5 +1,6 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
```

Comment:
We should delete this line, unless we're using this font ?

I thought we were using roboto

Reply:
I deleted this line. All that's left is to import roboto into the font config.